src/test/unit/algorithms/CondProbTableTest.cpp

```diff
@@ -180,13 +182,13 @@ namespace nupic {
 
 
   //----------------------------------------------------------------------
-  void CondProbTableTest::testTable(const string& testName, CondProbTable& table, 
-                                    const vector<vector<Real> > & rows)
+  void testTable(const string& testName, CondProbTable& table, 
```

Comment:
I'm not sure that assertions in this function will behave as intended. See here:
https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#propagating-fatal-failures